### PR TITLE
fix: next line issue with akhand paath mode

### PR DESCRIPTION
**Pull request by theamanjs**
Fixes issue #1505

### CODE REVIEW

**Comment by Gauravjeetsingh** [comment]
@theamanjs veerji
image

If I enable next line, and then enable akhand path view, I can see both settings turned on.

File: www/main/viewer/Slide/Slide.jsx [hunk 1]
```diff
@@ -140,7 +141,7 @@ const Slide = ({ verseObj, nextLineObj, isMiscSlide }) => {
                 order={transliterationOrder}
               />
             )}
-            {displayNextLine && nextLineObj && (
+            {!akhandpatt && displayNextLine && nextLineObj && (
```
Comment (Gauravjeetsingh) [inline www/main/viewer/Slide/Slide.jsx:144]: @theamanjs veerji instead of hiding the next line here, can we disable it from setting.

**Review by Gauravjeetsingh — CHANGES_REQUESTED** [review]

**Comment by Gauravjeetsingh** [comment]
@theamanjs veerji, I can take over changes in this PR.

File: www/main/viewer/Slide/Slide.jsx (hunk 1 above)
Comment (theamanjs) [reply www/main/viewer/Slide/Slide.jsx]: Veerji fixed

**Comment by theamanjs** [comment]
> @theamanjs veerji, I can take over changes in this PR.

Veerji, thank you for helping out.

**Review by Gauravjeetsingh — APPROVED** [review]

**Pull request merged**